versioned_docs/version-v1.4/upgrade/v1-4-2-to-v1-5-0.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 2
+sidebar_label: Upgrade from v1.4.2 to v1.5.0
+title: "Upgrade from v1.4.2 to v1.5.0"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.4/upgrade/v1-4-2-to-v1-5-0"/>
```

Comment:
1.5.0 will not be a supported upgrade version. For the upgrade, we should update the doc until the first stable version is ready, which will be 1.5.1.

Also, 1.5.0 is not released yet. 

cc @harvester/release